ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -408,6 +408,13 @@ export default defineComponent({
           setUnsavedChanges(false)
           // Delete the draft on successful submission
           if (getMhrInformation.value.draftNumber) await deleteMhrDraft(getMhrInformation.value.draftNumber)
+          const newItem: RegTableNewItemI = {
+            addedReg: String(getMhrInformation.value.mhrNumber),
```

Comment:
What about using `.toString()` ?

Reply:
good point...

Reply:
I did some testing with .toString() and it fails... 

Looked up the reason why and String(x) ensures the value of the object converted.
So I tried value.toString() and get a type error... I think String(x) is the most reliable...

Reply:
Hmm interesting why is it failing on type error, as `addedReg: string` is a string type.

Reply:
Why hard type coerce it at all? 
isn't `mhrNumber` typed and initialized as a string?

Reply:
`MhRegistrationSummaryIF`

Reply:
do you want me to change it?

Reply:
Is the code required/doing something or this is a decoy?

If the latter, then yes i'd just go with `addedReg:  getMhrInformation.value.mhrNumber`. Why cast it to a string if it already is, perhaps i'm missing something?

Reply:
Did some testing without String and ran into an issue...  I think we can get away without it, but there is another issue...I am going to fix..

Reply:
To answer your question..  This code is required for locating the current record saved.  I is used to ensure that the scrolling on the table locates the correct record...

Reply:
The code I was referring to was the casting of the value, not assigning the property value.

You are saying the `String()` is required? 
Seems like an underlying issue if we need to hard cast a string to a string..

Reply:
Made change..

Reply:
Resolved 👍